students/Jaidjen/lesson03/Mailroom1.py

```diff
@@ -0,0 +1,73 @@
+import sys  # imports go at the top of the file
+
+
+Mail  =[('James Smith',[2000.25, 500, 27.50]),
+        ('Dorothy Parker',[3175, 475.45, 92.30]),
+        ('Dohgyu Hwang',[10670, 350.19,175]),
+        ('Sally Yu', [100, 925]),
+        ('Jenny Ramos',[25.14])]
+
+prompt = "\n".join(("Functions:",
+          "Please choose from below options:",
+          "1 - Send a Thank You email",
+          "2 - Create a Report",
+          "3 - Quit",
+          ">>> "))
+
+
+def view_mail(Mail):
+    print("{:20s}{:20s}{:20s}{:20s}".format("DONOR ", "| TOTAL GIVEN ", "| NUMBER OF GIFTS ", "| AVERAGE GIFTS "))
+    print("-----------------------------------------------------------------------------")
+
+    for i in range(len(Mail)):
+        donate=[]
+        dnr_ave= sum(Mail[i][1])
+        ave_amt= int(len(Mail[i][1]))
+        tot_ave= dnr_ave/ave_amt
+        donate.append(dnr_ave)
+        sorted(donate)
```

Comment:
this is expensive, you are sorting each time you're looping - you're also sorting and not using sorted data (you need to store what is returned by `sorted` in a variable.